.travis.yml

```diff
@@ -12,6 +12,5 @@ env:
 
 before_install:
   - npm cache clear --force
-  - npm i npm@latest -g
```

Comment:
npm@latest fails on node 4, this must be recent but the whole point of this PR is to restore node 4.x compatibility to the the istanbul 1.x modules, I figured having the test work on that version is worth it.

Reply:
Well ... Isn't node 4 out of LTS already? I'd be ok with just removing it. We've been bitten in the past by keeping old node versions around too long.

Reply:
Yes, we have dropped node 4, but that was a breaking change intended to be for 2.x only.  A couple things happened wrong:
* istanbul-api@1.3.0 released to `next` upgraded to istanbul-lib-instrument@2.0.0.
* istanbul-lib-hook@1.2.0 released to `next` contained breaking API changes.

I never intended for istanbul-api@1.3.0 to be released to `latest`, but apparently `npm dist-tag add istanbul-api@2.0.5 latest` causes all versions `<=2.0.5` to be released to `latest`.  So yesterday istanbul-api@1.3.0 and istanbul-lib-hook@1.2.0 were released to `latest` on error.  The goal here is to re-release everything to higher 1.x versions as it was before any breaking changes.  This will be the final release for 1.x.

Reply:
Makes sense. Consider me resolved for now.